app/controllers/application_controller.rb

```diff
@@ -5,6 +5,16 @@ class ApplicationController < ActionController::Base
   responders :flash
   rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized
 
+  helper do
```

Comment:
@timurvafin, yep, missed that.

Another option is to have `CurrentUserHelper`:

```ruby
class ApplicationController
  helper :current_user
end
```

```ruby
module CurrentUserHelper
  def current_user
    UserPresenter.new(warden.authenticate(scope: :user))
  end
end
```